infrastructure/list-api/src/main.ts

```diff
@@ -412,8 +412,8 @@ class ListAPI extends TerraformStack {
           'arn:aws:iam::aws:policy/service-role/AmazonECSTaskExecutionRolePolicy',
       },
       autoscalingConfig: {
-        targetMinCapacity: config.environment === 'Prod' ? 6 : 1,
-        targetMaxCapacity: config.environment === 'Prod' ? 20 : 10,
+        targetMinCapacity: config.environment === 'Prod' ? 20 : 1,
+        targetMaxCapacity: config.environment === 'Prod' ? 100 : 10,
```

Comment:
Over the last 6 weeks list-api ran at 6 tasks almost constantly, with < 10% average CPU usage. Scaling it to a minimum of 20 tasks should give us room to handle a sudden spike in traffic.